### DISCUSSION TITLE
Is there a way to do conditionnal publishing?

### DISCUSSION THREAD

**Discussion by jimmydjabali** [codemagic-ci-cd/codemagic-docs#1172] [Q&A] [answered]
Hello,

Would it be possible to make a conditional publish to AppStore/Play Store ?

Like publish there only if `branch == "master"` or something ?

Thanks

**Comment by mikhail-tokarev** [comment, marked as answer]
Hi,

you should have different workflows for release and development pipelines. If you need extra security you can add a script to check that workflow is triggered for `master` branch
```
scripts:
    - if [ "$CM_BRANCH" != "master" ]; then exit 1; fi
```

or similar in post-clone script if you use Workflow Editor
